WebService/v1/Models/SimulationApiModel/SimulationStatistics.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models.SimulationApiModel
+{
+    public class SimulationStatistics
+    {
+        [JsonProperty(PropertyName = "TotalMessagesSent")]
+        public long TotalMessagesSent { get; set; }
+
+        [JsonProperty(PropertyName = "AverageMessagesPerSecond")]
+        public double AverageMessagesPerSecond { get; set; }
+
+        [JsonProperty(PropertyName = "FailedMessagesCount")]
+        public long FailedMessagesCount { get; set; }
+
+        [JsonProperty(PropertyName = "ActiveDevicesCount")]
+        public long ActiveDevicesCount { get; set; }
+
+        [JsonProperty(PropertyName = "FailedDeviceConnectionsCount")]
+        public long FailedDeviceConnectionsCount { get; set; }
+
+        [JsonProperty(PropertyName = "FailedDeviceTwinUpdatesCount")]
+        public long FailedDeviceTwinUpdatesCount { get; set; }
+
+        [JsonProperty(PropertyName = "SimulationErrorsCount")]
+        public long SimulationErrorsCount { get; set; }
+
+        // Default constructor used by web service requests
+        public SimulationStatistics()
+        {
+            this.TotalMessagesSent = 0;
+            this.AverageMessagesPerSecond = 0;
+            this.FailedMessagesCount = 0;
+            this.FailedDeviceConnectionsCount = 0;
+            this.FailedDeviceTwinUpdatesCount = 0;
+            this.SimulationErrorsCount = 0;
+        }
+
+        // Map API model to service model
+        public static Services.Models.Simulation.StatisticsRef ToServiceModel(SimulationStatistics statistics)
+        {
+            if(statistics != null)
+            {
+                return new Services.Models.Simulation.StatisticsRef {
```

Comment:
we'll have to revisit the class names, e.g. the idea about classes with a "Ref" suffix was to be a subset of the complete class, i.e. a reference to the actual object. I think this is the second instance where that idea is not applied, so at this point I'm not sure what "Ref" means anymore :-)  We'll see this separately once we're done with major tasks, not a blocker because it doesn't impact clients

Reply:
This StatisticsRef is a subclass, it only contains 2 properties instead of all 5